.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+
+cache:
+  directories:
+    - "node_modules"
+
```

Comment:
I think we should defined some node version instead of always using the stable one?
At least `6.10` the one running in Vega